lib/bloc/account/fiat_conversion.dart

```diff
@@ -53,11 +58,35 @@ class FiatConversion {
     // if conversion result is less than the minimum it doesn't make sense to display
     // it.
     if (!allowBelowMin && fiatAmount < minimumAmount) {
-      formattedAmount += minimumAmount.toStringAsFixed(fractionSize);
+      formattedAmount = minimumAmount.toStringAsFixed(fractionSize);
       prefix = '< ' + prefix;
     } else {
-      // Otherwise just show the formatted value.
-      formattedAmount += fiatAmount.toStringAsFixed(fractionSize);
+      if (useBlankGroupSeparator) {
```

Comment:
What is the reason we need a parameter? Shouldn't we be consistent and add the blank separator on all formatting except for user input (like sats) ?

Reply:
There were elements not mentioned in the PR, such as exchange rate label on currency converter label. I assumed these changes only applied to POS fields.
Removed this parameter so the formatting is consistent throughout the app.